src/server.js

```diff
@@ -0,0 +1,34 @@
+import 'dotenv/config'
+import { WebSocketServer } from 'ws'
+import ZmqConnection from './zmq.js'
+import Client from './client.js'
+import Dashcore from '@dashevo/dashcore-lib'
+
+export default class Server {
+  #clients = []
+
+  constructor () {
+    this.zmq = new ZmqConnection()
+    this.zmq.on('rawtx', (hex) => this.handleRawTx(hex))
+  }
+
+  async start () {
+    const socket = new WebSocketServer({ port: process.env.PORT })
+
+    socket.on('connection', (socket) => {
+      const client = new Client(socket)
+      this.#clients.push(client)
+      client.sendMessage('You are successfully connected to the Dash WebSocket server!')
+    })
+
+    const addressInfo = socket.address()
+    const host = addressInfo.address === '::' ? 'localhost' : addressInfo.address
```

Comment:
I think this is not need, it is showing what address what exactly ip interface it was bound on.

Actually. that basically depends on what your port your application was bound on. I think you need an environment variable to set the actual IP address that you want to bind on, in case you wanna change it.
It should show correctly in this case.

Such variable will be also necessary if you want to bind on 0.0.0.0 and share the access to your service

Reply:
This is also important not to change output of this address so it is more clear for devops that do debug their deployments